about.css

```diff
@@ -0,0 +1,388 @@
+}
+
+.mobile {
+  list-style-type: none;
+}
+
+.closeMenu {
+  float: right;
+  margin-right: 20px;
+  font-size: 30px;
+  cursor: pointer;
+  color: red;
+}
+
+.navSlide {
+  display: none;
+}
+
+.phrases {
+  margin-top: 20px;
+  margin-left: 30px;
+  color: white;
+  text-align: center;
+}
+
+.phrasesh2 {
+  font-size: 25px;
+}
+
+.phraseh1 {
+  font-size: 34px;
+}
+
+.headParaDiv {
+  margin-left: 30px;
+  border: 1px solid red;
+  background-color: white;
+  opacity: 0.6;
+  margin-right: 30px;
+}
+
+.headPara {
+  margin-left: 5px;
+  color: red;
+  text-align: center;
+}
+
+.headLoc {
+  margin-left: 30px;
+  color: black;
+  text-align: center;
+  margin-top: 30px;
+}
+
+.olugI {
+  height: 90px;
+  margin-top: 20px;
+}
+
+.plugDetail {
+  display: flex;
+  flex-direction: column;
+}
+
+.plugEst {
+  display: flex;
+  justify-content: center;
+  border: 1px solid red;
+  margin-left: 30px;
+  margin-right: 30px;
+  margin-top: 50px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.plgDetPara {
+  text-align: center;
+  margin-top: 30px;
+  font-size: 30px;
+}
+
+.plugDetHr {
+  margin-left: 150px;
+  margin-right: 150px;
+  color: red;
+}
+
+.valuePara {
+  text-align: center;
+  color: rgb(87, 88, 88);
+}
+
+.plugEsth {
+  font-size: 20px;
+}
+
+.plugEsDat {
+  border-bottom: 0.5px solid rgb(87, 88, 88);
+  border-top: 0.5px solid rgb(87, 88, 88);
+  font-size: 10px;
+  text-align: center;
+  margin-top: -30px;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+  border-top: 1px solid rgb(87, 88, 88);
+  margin-top: 50px;
+}
+
+.projectH {
+  text-align: center;
+  font-size: 30px;
+}
+
+.projectHr {
+  color: red;
+  margin-left: 150px;
+  margin-right: 150px;
+  margin-top: -20px;
+}
+
+.projectPara {
+  text-align: center;
+  color: rgb(143, 147, 150);
+}
+
+.projectBack {
+  background-color: red;
+  opacity: 0.5;
+  height: 200px;
+  margin-top: -20px;
+  color: white;
+}
+
+.project1 {
+  background-image: url("img/project1.jpg");
+  background-size: cover;
+  text-align: center;
+  height: 200px;
+  width: 350px;
+}
+
+.project2 {
+  background-image: url("img/project2.jpg");
+  background-size: cover;
+  text-align: center;
+  width: 350px;
+  height: 200px;
+}
+
+.projectDet {
+  display: grid;
+  grid-template-columns: auto;
+  grid-gap: 20px;
+  justify-content: center;
+}
+
+footer {
+  margin-top: 100px;
+}
+
+.partners {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(58, 56, 56);
+  padding: 10px;
+}
+
+.partnersDiv {
+  text-align: center;
+  padding: 20px 0;
+  font-size: 30px;
+}
+
+.partOn {
+  display: flex;
+  justify-content: center;
+  gap: 80px;
+}
+
+.partTw {
+  justify-content: center;
+  gap: 80px;
+  display: flex;
+}
+
+.Div1 {
+  color: white;
+}
+
+.Div1Hr {
+  color: red;
+  margin-left: 150px;
+  margin-right: 150px;
+}
+
+.Div2 {
+  color: rgb(159, 162, 163);
+}
+
+.Div4 {
+  color: rgb(142, 145, 148);
+}
+
+.Div5 {
+  color: rgb(190, 195, 199);
+}
```

Comment:
- [ ] There are still some class names that start with uppercase letters please, make them lower case as the previous reviewer suggested.